tests/subscription/make-payment-failure.spec.js

```diff
@@ -0,0 +1,14 @@
+// @ts-check
+const { test, expect } = require('@playwright/test');
+
+test('Make Payment (failure)', async ({ page }) => {
+    // call to invalid token
+    await page.goto('/admin/makepayment/12345');
+
+    // check error message
+    await expect(page.locator('text="Payment failed for RecurringDetailReference"')).toBeVisible();
+    // check error icon
+    await expect(page.locator('.status-image >> visible=true')).toBeVisible();
```

Comment:
This can be solved having a different `css` class for the `success` and `error` icons (at the moment it is the same).
Alternatively we check an image is displayed, and we check the error message (see your previous comment)